### ISSUE TITLE
White paper 'How can I make my Information more Accessible?'

### ISSUE THREAD

**Issue by SaraRydin** [SaraRydin/srydinsord#9]

**Comment by SaraRydin** [comment]
### Summary and purpose
You have a client who requests that you adapt your information to make it more accessible. Working with accessibility can be tricky if you don't have the habit of doing so. The purpose of the whitepaper is to show you how to make your information accessible, given your client's needs.

This paper provides a basic understanding of accessibility and guidance on where to find more information if needed.

Accessibility of information involves two aspects. First, a person must be able to access information. Secondly, accessibility also requires an understanding of information.
In the paper, you will learn about

- U.S. and European electronic standards for documents
- Assistive technology and how they affect accessibility
- Different formats
- Some design principles

 
### Information is more than text
Information is not equivalent to text. However, there is much more to information, including images, sounds, and moving images. 

Remember this when talking to a new customer who wants accessibility since some prefer video, while others prefer plain text. For example, you might favor other formats if your reading ability is impaired. Please do not assume anything is a given but ask what format/formats they prefer.

### What is accessibility?
There are many aspects of accessibility, for instance, when creating information, services, and buildings. Nevertheless, this paper focuses only on information accessibility.

[16% of the world's population is estimated to be disabled according to WHO](https://www.who.int/health-topics/disability#tab=tab_1s/detail/disability-and-health), ranging from mobility to cognitive impairment. The remaining 84% may also experience a temporary disability, according to the same source, such as breaking the dominant arm or losing the voice for a few days, so they cannot move or speak at some point.

Situations can also disable any person, such as when they have the sun in their eyes and cannot see the screen or when they are underground and cannot access the Internet.

Accessibility is logical and also beneficial to all users. If you design for all ability levels, you can create products and services anyone can use and enjoy. Therefore, you can reach more users on several types of machines.
 
### Customization and standards
Many governmental agencies require you to follow a standard due to legislation that requires accessibility. Additionally, there are standards used unrelated to legislation. 
However, not everything follows standards yet, and cognitive accessibility, for instance, is not covered by most standards. Consider this when discussing your customer needs and what standard to follow. 

In many cases, your customer wants to adhere to a specific standard. U.S. and European accessibility standards include:

- Section 508—is part of The Rehabilitation Act under U.S. law. On this site, you can find guidance for information accessibility. 
- EN 301 549—is a European standard for digital accessibility. Version 2.1.2 references WCAG 2.1 AA but covers more.
- The Web Accessibility Directive—is a directive on the accessibility of websites and mobile applications adopted by the E.U. It refers, among other standards, to the EN 301 549 standard. 
- WCAG 2.1 levels A-AAA—The Web Content Accessibility Guidelines is an international standard for web content accessibility developed by The World Wide Web Consortium (W3C). WCAG 2.0 is equivalent to ISO/IEC 40500. 
- PDF/UA (ISO 14289)—is a standard defining how to make the PDF format accessible. 
- ADA Compliance (ADA = Americans with Disabilities Act)— refers to the text of the ADA and other similar disability regulations. 

[Information about accessible publishing at the publications office of the European Union. ](https://op.europa.eu/en/web/accessibility/documents)
[More information about U.S. regulations and laws at The U.S. Access Board. ](https://www.access-board.gov/)
 
### Adjust for Assistive Technology
Some people with disabilities have assistive technology, meaning your information must work with their technology. Others have never received personal assistive technology, so you must customize it to meet their needs.
Among the most common assistive technologies are:

- Screen readers that enable people with visual impairments or who are blind to read the information on the screen (links, headings, texts, tables). Screen readers read the text and other significant elements aloud.

- Braille readers that enable blind or visually impaired persons to convert text into braille characters in the reader.

- Text-to-speech tools that help people with reading disabilities to hear all or select parts of the text on the screen.

Two things to remember here are:

- Those who cannot see might use only a keyboard. Therefore, it should be possible to tab through the screen. A person can navigate between menus, headings, and paragraphs by pressing the tab key on the keyboard.

- You must have used the correct characters when the speech synthesis or screen reader reads your text. For example, - (minus), - (hyphen), and – (en dash) are different signs and are therefore pronounced in different ways.

 
### Document format
Your customer may know which format suits them, but their assistive technology may also work best with a specific format. Read more in the paragraph "Adjust for assistive technology?".

If intended for the general public, adjust the information for lack of assistive technology.

#### Microsoft Word
Microsoft Word is one of the most widely used document tools and formats. You cannot assume the file format is accessible to the public since it is a commercial tool.

Today, Word includes helpful accessibility controls that help you create accessible documents. You can use the accessibility control to add alternative text to images and mark up tables, among other things.

#### PDF
Free reading tools are one of the advantages of PDF as a document format. 
You can create the PDF in two ways:

- From a source document created with Microsoft Word, Google Docs, or Adobe InDesign
- By scanning a physical document

As to the latter PDF files, they are usually just pictures and are not accessible at all. 

You can make the former PDF accessible by tagging the content to identify headings, lists, etc., and adding alternate text for images. You must also set the logical reading order. Assistive technology uses all these features to access PDF content. 

Adobe has a tool that you can use to check for full accessibility.

You can also [read about creating accessible PDFs under Section 508 of U.S. law at](http://www.section508.gov/) [www.section508.gov](http://www.section508.gov/).
See also the [section about accessibility in Adobe's user guide](https://helpx.adobe.com/support/acrobat.html).

#### HTML
WordPress has become increasingly popular for creating websites over the last few years.
You can find code standards for accessibility on the WordPress website. There you will learn how to mark up your site and make your content accessible.

You can find WordPress themes that are accessible, as accessibility has become legally required. As a result, you do not have to do everything yourself but can partly rely on an existing theme.

Note that 

- You should use the aria-label/aria-labelledby attribute for interactive elements.

- The pronunciation of different screen readers and speech synthesizers might differ. Test some to see in what way — especially for non-standard language.

[Read more about aria-label and aria-labelledby on the W3 page.](https://www.w3.org/TR/wai-aria/#aria-label)
 
### Design principles
We will discuss different aspects of information and how you can make it accessible.

#### Provide the reader with alternatives
There should always be at least two ways to absorb information. For example, you could offer the following

Both text and text-to-speech reading of the text
Both video and the transcribed text
Keep in mind that you have to provide all non-text content with text. Symbols and images, for example, must be described. See further under the chapter Make image accessible.

#### Consistent och structured
Many disabilities come with cognitive challenges. Consistent and predictable information is the best support you can give those people. It should also be easy to navigate, find specific information, and find your way around.

Keep these things in mind:

- Ensure that you write your information so that everyone can understand it without assistance from teachers or others. Communicate in plain or easy-to-understand language.
- Maintain consistency in the presentation of information of the same type.
- Give those who need access to enhanced background knowledge. You could use a glossary for technical terms.
- Provide the most relevant information first.
- Delete all unnecessary information.
- Provide extra clarity when the reader needs to make a decision.
- Make sure you create headings at the correct level. At level one (h1), there should never be more than one heading.
- Ensure the information source is as robust as possible for future assistive technology.

Read more about cognitive accessibility [in the material by the Coga research group on the World Wide Web Consortium.](http://www.w3.org/TR/coga-user-research/)

#### Time and cognitive load
A well-designed information presentation gives the working memory enough time to process words, sounds, and images. More complex information ought to have a lower cognitive load. Also, you should avoid creating unnecessary mental burdens by placing time constraints on readers.

Different people have different processing capacities, partly determined by

- The reader's level of expertise
- The reader's age
- If the reader has problems with their working memory or other processes that affect it

#### Make text accessible
Here are some tips on how to organize information, write links, and how you can enable the reader to choose fonts. 

##### Organize information
We do not read from the first to last letter in non-fictional texts. Our eyes scan the page for places to start reading and jump from one paragraph to the next. (The same cannot be said for fiction.)

It is best to begin the text with a summary so poor or slow readers will know what it is about. 

Further, guide the reader by 

- Keeping paragraphs short
- Starting the paragraph with the essential information first
- Including subheadings when appropriate

A natural place to begin reading is at the beginning of the paragraph or the subheading. 
Add lists to make the text easier to understand. Although lists are good, they should be short. Try instead to divide the information into groups and make a list of each group. 

Also, divide information in tables into smaller groups to make it easier to understand. 
The final step is to add space to your text so the reader can follow and understand it.

##### Links
If the text contains a link, tell the reader what happens when they click on it and where they end up. For example:

- [I always watch cat movies on Youtube](https://www.youtube.com/watch?v=cbP2N1BQdYc)
- [Download the manual as a PDF](https://helpx.adobe.com/se/acrobat/user-guide.html)

In the latter case, if you use a screen reader, you hear "link" and then "download the manual as a PDF."

##### Fonts
Enable the reader to:

- Choose from various commonly used fonts and sizes. (There is no single accessible font for all people. It's just a sales gimmick.)
- Increase the space between individual letters in the font. It increases accessibility.

For more tips on accessible text, see[ the World Wide Webb Consortium's guide on accessible text.](https://www.w3.org/WAI/tips/writing/)

#### Make video and audio accessible
Factors influencing video accessibility are, among others:

- The amount of information displayed is not too large.
- The time given to read the subtitles is generous.
- Transcript is always available.
- The pace for audio or audio description is not set too high. However, allow readers to speed up the reading if they wish.

One of the most important adaptations of sound is to make sure to avoid background noise while someone is speaking. 
[Read the EASIT project reports on webs.uab.cat](https://webs.uab.cat/easit/en/) for more information. In the reports, you will find information on how to make subtitles that are easy to understand, audio descriptions of what is happening in the picture, and news that is easy to understand.

See also[ the audio and video media guide from World Wide Web Consortium.](https://www.w3.org/WAI/media/av/)

#### Make images accessible
Describe all images for those who cannot see them. What is relevant to describe naturally depends on the context. 
If the image is decorative, add the description "" to the image so that screen readers can ignore it.
